ERROR Could not pass event NPCRightClickEvent to Quests v2.7.3-b116

31.08 13:41:42 [Server] ERROR Could not pass event NPCRightClickEvent to Quests v2.7.3-b116
31.08 13:41:42 [Server] INFO org.bukkit.event.EventException
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:310) ~[spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.citizensnpcs.EventListen.onPlayerInteractEntity(EventListen.java:385) [Citizens.jar:?]
31.08 13:41:42 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method) ~[?:1.8.0_91]
31.08 13:41:42 [Server] INFO at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62) ~[?:1.8.0_91]
31.08 13:41:42 [Server] INFO at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_91]
31.08 13:41:42 [Server] INFO at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_91]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnection.a(PlayerConnection.java:1315) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInUseEntity.a(SourceFile:52) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.PacketPlayInUseEntity.a(SourceFile:11) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.PlayerConnectionUtils$1.run(SourceFile:13) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_91]
31.08 13:41:42 [Server] INFO at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_91]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.SystemUtils.a(SourceFile:44) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:715) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:374) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:654) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:557) [spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at java.lang.Thread.run(Thread.java:745) [?:1.8.0_91]
31.08 13:41:42 [Server] INFO Caused by: java.lang.NullPointerException
31.08 13:41:42 [Server] INFO at org.bukkit.inventory.ItemStack.<init>(ItemStack.java:68) ~[spigot.jar:git-Spigot-e4d4710-e1ebe52]
31.08 13:41:42 [Server] INFO at me.blackvein.quests.Quester.deliverItem(Quester.java:1411) ~[?:?]

My quests.yml:

> quests:
  Beggar:
    name: Beggar
    npc-giver-id: 6
    ask-message: <yellow>I am so hungry, please find me a cooked chicken to eat.
    finish-message: <grey>The beggar eats well tonight. Quest complete.
    stages:
      ordered:
        '1':
          items-to-deliver:
          - name-wood_pickaxe:amount-1
          npc-delivery-ids:
          - 6
          delivery-messages:
          - "<npc>: Keep it up!"
    rewards:
      money: 1000
      quest-points: 1

--- a/src/main/java/me/blackvein/quests/Quester.java
+++ b/src/main/java/me/blackvein/quests/Quester.java
@@ -1391,7 +1391,8 @@ public class Quester {
 
             int amount = getQuestData(quest).itemsDelivered.get(found);
             int req = getCurrentStage(quest).itemsToDeliver.get(getCurrentStage(quest).itemsToDeliver.indexOf(found)).getAmount();
-Material m = null;
+            Material m = i.getType();
+
             if (amount < req) {
 
                 if ((i.getAmount() + amount) > req) {

